### PR TITLE
rbd: cleanup code and output formatting in status action

### PR DESCRIPTION
**Pull request by dillaman**
Signed-off-by: Jason Dillaman <dillaman@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by dillaman** [comment]
jenkins test make check

**Comment by dillaman** [comment]
jenkins test make check

File: src/tools/rbd/action/Status.cc [hunk 1]
```diff
@@ -217,11 +217,12 @@ static int do_show_status(librados::IoCtx& io_ctx, const std::string &image_name
...
-        std::cout << "image cache state:" << std::endl;
+        std::cout << "Image cache state:" << std::endl;
         std::cout << "\tclean: " << (cache_state.clean ? "true" : "false")
-                  << "  size: " << byte_u_t(cache_state.size)
-                  << "  host: " << cache_state.host
-                  << "  path: "  << cache_state.path << std::endl;
+                  << std::endl
+                  << "\tsize: " << byte_u_t(cache_state.size) << std::endl
+                  << "\thost: " << cache_state.host << std::endl
+                  << "\tpath: "  << cache_state.path << std::endl;
```
Comment (trociny) [inline src/tools/rbd/action/Status.cc:225]: nit: while here could you also remove the extra space between `"\tpath: "` and `<<`?

**Review by trociny — APPROVED** [review]
LGTM

**Pull request merged**